src/http-error-dialog/index.tsx

```diff
@@ -26,13 +65,14 @@ export const HttpErrorDialog: FC<HttpErrorDialogProps> = ({
   const [detailsShown, toggleDetailsShown] = useState(false);
   const detailsRef = useRef<HTMLDivElement>(null);
 
-  const code = error.code ?? 'Unknown';
-  const details = error.msg.length > 0 ? error.msg : 'No details available';
+  const code = error.statusCode ?? 'Unknown';
+  const details =
+    error.message.length > 0 ? error.message : 'No details available';
   const type =
     code >= 400 && code < 500
-      ? 'Client'
+      ? 'client'
```

Comment:
Avoid nested ternary (I just setup ESLint and that's one of the errors defined in the `airbnb-typescript` config).